cli/cmd/service.go

```diff
@@ -286,6 +286,43 @@ func (c *ServicedCli) printServicesAll(ctx *cli.Context) {
 	}
 }
 
+func (c *ServicedCli) printHelpForRun(svc *service.Service, command string) (returncode int) {
+	found := false
+	availablecommands := make([]string, 0, len(svc.Commands))
+	sort.Strings(availablecommands)
+	for commandname := range svc.Commands {
+		availablecommands = append(availablecommands, commandname)
+		if commandname == command {
+			found = true
+		}
+	}
+
+	if command == "help" {
+		fmt.Printf("Available commands for %v:\n", svc.Name)
+		for _, commandname := range availablecommands {
+			fmt.Printf("    %-20v  %v\n", commandname, svc.Commands[commandname].Description)
+		}
+		if len(availablecommands) == 0 {
+			fmt.Println("    No commands available.")
+		}
+		return 0
+
+	} else if !found {
+		fmt.Printf("Command %#v not available.\n", command)
+		fmt.Printf("Available commands for %v:\n", svc.Name)
+		for _, commandname := range availablecommands {
+			fmt.Printf("    %-20v  %v\n", commandname, svc.Commands[commandname].Description)
+		}
+		if len(availablecommands) == 0 {
+			fmt.Println("    No commands available.")
+		}
+		return 1
+
+	} else {
```

Comment:
Go style prefers that you simply end the if block here and return -1 in the function scope.